### ISSUE TITLE
Unhandled exception when opening Setup window for an entry with a seed, but no settings

### ISSUE THREAD

**Issue by robinvanpoppel** [KeeTrayTOTP/KeeTrayTOTP#133]
**Steps to reproduce:**

* Open an entry with valid TOTP settings via Keepass' "Edit settings" (F2)
* Go to the Advanced tab, remove the "TOTP Settings", and click OK
* Open "Setup TOTP".

**Result:**
Crashes with a NullReferenceException in `SettingsGet`:

**Expected:**
At least no unhandled exception (which crashes Keepass). Maybe showing some default settings, or at least a prompt asking to apply these.

Stacktrace:
```
KeeTrayTOTP.dll!KeeTrayTOTP.KeeTrayTOTPExt.SettingsGet(KeePassLib.PwEntry pe) Line 783
	at C:\Git\KeeTrayTOTP\KeeTrayTOTP\TrayTOTP_Plugin.cs(783)
KeeTrayTOTP.dll!KeeTrayTOTP.SetupTOTP.SetupTOTP_Load(object sender, System.EventArgs e) Line 40
	at C:\Git\KeeTrayTOTP\KeeTrayTOTP\SetupTOTP.cs(40)
[External Code]
KeeTrayTOTP.dll!KeeTrayTOTP.KeeTrayTOTPExt.OnEntryMenuSetupClick(object sender, System.EventArgs e) Line 369
	at C:\Git\KeeTrayTOTP\KeeTrayTOTP\TrayTOTP_Plugin.cs(369)
```

**Comment by dannoe** [comment]
I can not reproduce this with the current master branch.
This was fixed with #148, wasn't it?

**Comment by robinvanpoppel** [comment]
I think you're right. Let's close this one.
